packages/@okta/vuepress-site/guides/sign-into-mobile-app/configure-packages/android/configuremid.md

```diff
@@ -1,10 +1,20 @@
-Create a new `okta_oidc_config.json` file in your application's `res/raw` with the following contents:
+Create a configuration object in code:
+
+```java
+account = new OIDCAccount.Builder()
+    .clientId("{clientId}")
+    .redirectUri("{redirectUri}")
+    .scopes("openid", "profile", "offline_access")
+    .discoveryUri("https://{yourOktaDomain}")
```

Comment:
@FeiChen-okta  Is there a way to point the SDK at the `/oauth2/default` Authorization Server? (see the iOS instructions for comparison)

Reply:
Yes. In the discoveryUri they can put `https://{yourOktaDomain}`for OIDC metadata.
or `https://{yourOktaDomain}/oauth2/${authServerId}` for OAuth2 server metadata. The SDK appends the correct `/.well-known//openid-configuration` or `.well-known/oauth-authorization-server`